articles/connections/social/auth0-oidc.md

```diff
@@ -2,23 +2,29 @@
 connection: Auth0 OpenIDConnect
 seo_alias: auth0-oidc
 image: /media/connections/auth0.png
-description: You can use a Client on another Auth0 tenant as an identity provider in your current Auth0 tenant.
+description: You can use a Client on another Auth0 tenant as an OIDC identity provider in your current Auth0 tenant.
 toc: true
 ---
 
 # Authenticate using OpenIDConnect to another Auth0 Tenant
 
-You can use a Client on another Auth0 tenant (referred to below as the **child tenant**) as an identity provider in your current Auth0 tenant (the **master tenant**).
+You can use a client on another Auth0 tenant (referred to below as the **OIDC Provider tenant**) as an identity provider in your current Auth0 tenant (the **Relying Party tenant**).
 
-## Configure the Child Auth0 Tenant
+## Configure the OIDC Provider Auth0 Tenant
 
-1. Create a Client or edit an existing one.
-2. Take note of its **clientID** and **clientSecret**. You will need these to create the connection in the master tenant.
-3. Add the master tenant's login callback to the list of **Allowed Callback URLs**: `https://${account.namespace}/login/callback`
+1. Create a Client or edit an existing one. Set the client type to regular web app.
+2. Take note of its **clientID** and **clientSecret**. You will need these to create the connection in the Relying Party tenant.
+3. Add the Relying Party tenant's login callback to the list of **Allowed Callback URLs**: `https://${account.namespace}/login/callback`
 
 ![](/media/articles/connections/social/auth0-oidc/child-app.png)
 
-## Configure the Master Auth0 Tenant
+4. Ensure that the **OIDC-Conformant** toggle in the **OAuth** tab under the client's **Advance Settings** is turned **off**.
```

Comment:
I have tested while **OIDC-Conformant** toggle turned on for the client in OIDC Provider in a test tenant. I can still authenticate the user and get the tokens. May be this is already fixed now?

Reply:
What id did the user got? I don't think it's fixed yet.

Reply:
Ah, yes it is undefined, just noticed it. `auth0-oidc|SALTUK-OIDC-CHILD-CON|undefined`. Please disregard this.